app/pods/cases/case/subcases/overview/controller.js

```diff
@@ -1,18 +1,22 @@
 import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency-decorators';
 
-export default Controller.extend({
-  queryParams: ['refresh'],
 
-  isShowingOptions: false,
-  isShowingProcess: false,
-  refresh: false,
+export default class CaseSubcasesOverviewController extends Controller {
+  @service toaster;
+  @service intl;
 
-  actions: {
-    backLinkAction() {
-      this.transitionToRoute('cases');
-    },
-    refresh() {
-      this.toggleProperty('refresh');
-    },
-  },
-});
+  @task
+  *saveSubcase(subcase) {
```

Comment:
Deze zou ik eerder `addSubcase` noemen (en het argument van de component dan `onAddSubcase`). Dat verklaart beter waarom er hier een `reloadModel` nodig is. Wanneer een bestaande subcase gewijzigd wordt (wat `saveSubcase` impliceert) zou het model niet herladen moeten worden.